src/main/java/com/inturn/pfit/domain/size/dto/request/CreateSizeRequestDTO.java

```diff
@@ -0,0 +1,59 @@
+package com.inturn.pfit.domain.size.dto.request;
+
+import com.inturn.pfit.domain.size.entity.SizeEntity;
+import com.inturn.pfit.domain.sizetype.dto.request.CreateSizeTypeRequestDTO;
+import com.inturn.pfit.domain.sizetype.entity.SizeTypeEntity;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Builder;
+import org.hibernate.validator.constraints.Length;
+import org.springframework.util.CollectionUtils;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Builder
+public record CreateSizeRequestDTO(
+
+		@NotNull
+		Integer categoryId,
+
+		@NotEmpty
+		@Length(max = 255)
+		String sizeName,
+
+		@Valid
+		@Size(min = 1)
+		@NotNull
+		List<CreateSizeTypeRequestDTO> sizeTypeList
+
+) {
+
+	public SizeEntity createSize() {
```

Comment:
```suggestion
	public SizeEntity convertSize() {
```

새로 만드는게 아니라 객체가 가지고 있는걸 변환한다의 의미로 보여서요 ㅎ

Reply:
네! 비슷한 메소드도 수정하도록 하겠습니다.